### ISSUE TITLE
C-nano client #include uses krpc_cnano directory instead of krpc

### ISSUE THREAD

**Issue by VassilyDev** [Genhis/KRPC.MechJeb#11]
Are there versions compatible with KSP 1.5.0?
Thanks.

**Comment by Genhis** [comment]
Hello, you can try [v0.3.0](https://github.com/Genhis/KRPC.MechJeb/releases/tag/v0.3.0).
If you can get kRPC 0.4.8 and MechJeb 2.8.3.0 working on KSP 1.5, you can download [v0.5.1](https://github.com/Genhis/KRPC.MechJeb/releases/tag/v0.5.1).

**Comment by VassilyDev** [comment]
Thank you. I can confirm that the combination of KSP 1.5, kRPC 0.4.8, MJ 2.8.3.0 and kRPC.MechKeb seems to work without any issue.

By the way, I did not really understand how to integrate the C-nano client, I had to rename the file to be a .h header instead of the provided c file. I had to change and integrate the #include statements as well, because the library of kRPC downloaded from Arduino Library Manager is named kRPC and not kRPC_nano. Here is the modified part:
```
#include <krpc.h>
#include <krpc/services/krpc.h>
#include <krpc/services/space_center.h>
#include <krpc/decoder.h>
#include <krpc/encoder.h>
#include <krpc/error.h>
#include <krpc/memory.h>
#include <krpc/pb_decode.h>
#include <krpc/pb_encode.h>
#include <krpc/types.h>
```

**Comment by Genhis** [comment]
> I had to change and integrate the #include statements as well, because the library of kRPC downloaded from Arduino Library Manager is named kRPC and not kRPC_nano.

The client code is automatically generated. C-nano client in [kRPC github repository](https://github.com/krpc/krpc/releases) uses `krpc_cnano` directory for services and other header files - I don't know why Arduino Library Manager doesn't. I hope it's not a big issue to rename those include statements.

**Comment by Genhis** [comment]
kRPC fixed this issue for v0.5.2.
